src/style/aside.scss

```diff
@@ -0,0 +1,23 @@
+@use './spacing';
+@use './colors';
+
+aside {
+    &.inline {
+        @include spacing.h-margins(4em);
```

Comment:
This goes against the _Cascading_ part of CSS.

It's small enough though and only used in 1 place. I'll just leave it be.